source/frontend/src/models/api/AcquisitionFile.ts

```diff
@@ -11,6 +11,7 @@ export interface Api_AcquisitionFile extends Api_ConcurrentVersion, Api_AuditFie
   id?: number;
   fileNo?: number;
   ministryProjectNumber?: string;
+  legacyFileNumber?: string;
```

Comment:
legacyFileNumber is a **long** in the backend model while here is a string - please fix one or the other